### PR TITLE
Ex1

### PR DESCRIPTION
**Pull request by bytesontherocks**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

**Comment by CorndelDevOps** [comment]
loving the username

File: todo_app/app.py [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
+from flask import Flask, redirect, url_for, render_template, request
+from todo_app.data.session_items import get_items,add_item
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
-@app.route('/')
+@app.route('/index')
```
Comment (CorndelDevOps) [inline todo_app/app.py]: Currently it gives an error when I go to the base URL, which is weird.

You will want to have something at '/'. You could make  '/' to redirect to '/index', but having it as '/' makes the index URL clean.

File: todo_app/templates/index.html [hunk 2]
```diff
@@ -4,15 +4,26 @@
...
+    <p class="lead"></p>ByteOnTheRocks to-do app: So much to do, wake up!</p>
   </div>
 
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}
+          <li>{{item}}</li>
```
Comment (CorndelDevOps) [inline todo_app/templates/index.html:15]: You could render these items more elegantly. You could but don't have to do it now. You will at one point.

`{{ c.id }}` will give you the id value in HTML, for example and you can build slightly more readable, slick UI if you tinker with HTML and CSS a little bit.

image

**Review by CorndelDevOps — APPROVED** [review]
Nice job with your first submission. It looks all good.

Left some pointers for getting things rendering/rendering more pretty. Have a go at them if you want before completing the PR.

File: todo_app/app.py (hunk 1 above)
Comment (bytesontherocks) [reply todo_app/app.py]: amended

**Pull request merged**